omnibus/config/software/private-chef-cookbooks.rb

```diff
@@ -29,7 +29,6 @@
       " -v 2.1.3" \
       " --no-rdoc --no-ri", env: env
 
-
```

Comment:
We have a berkshelf based license collector in the works too so we need to leave this definition alone for now 😄